expo/gbe/functions.py

```diff
@@ -145,3 +146,24 @@ def available_volunteers(event_start_time):
         if event_start_time in window_range:
             windows.append(window)
     return conf.Volunteer.objects.filter(available_windows__in=windows)
+
+
+def get_events_and_windows(conference):
```

Comment:
Surely a function which queries a table in the scheduler app should be under the scheduler app, no?